docs/howto/deploy/aws.md

```diff
@@ -176,13 +176,18 @@ Checkout Nginx [documentation](https://kubernetes.github.io/ingress-nginx/user-g
 
 1. From the S3 Administration console, choose _Create Bucket_.
 2. Use the following as your bucket policy, filling in the placeholders:
+
+    **Note:** For Amazon S3 Express One Zone storage class with directory buckets, apply the Standard Permission with s3express.
```

Comment:
I would simply change (1) to "Take note of the bucket name you want to use with lakeFS".